src/components/UI/NavigationBar/DrawerFooter.tsx

```diff
@@ -0,0 +1,41 @@
+import SportsBarIcon from '@mui/icons-material/SportsBar'
+import TopicIcon from '@mui/icons-material/Topic'
+import {
+  Divider,
+  ListItemIcon,
+  ListItemText,
+  MenuItem,
+  MenuList as MuiMenuList,
+} from '@mui/material'
+import useConfig from 'hooks/useConfig'
+import useVersion from 'hooks/useVersion'
+
+const DrawerFooter = () => {
+  const config = useConfig()
```

Comment:
See other comment for context.

Is extracting the config into its own hook necessary?

Would
```typescript
const { config } = ServerConfigContainer.useContainer()
```
work just as well? Or is there something I'm not seeing?